src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+        Todo todo = new Todo(description);
+        inputs[index] = todo;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + todo
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createEvent(String line){
+        String removedCommand = line.replace("event", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/at")).strip();
+        String date = removedCommand.substring(removedCommand.indexOf("/at") + "/at".length()).strip();
+        Event event = new Event(description, date);
+        inputs[index] = event;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + event
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createDeadline(String line) {
+        String removedCommand = line.replace("deadline", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/by")).strip();
+        String date = removedCommand.substring(removedCommand.indexOf("/by") + "/by".length()).strip();
+        Deadline deadline = new Deadline(description, date);
+        inputs[index] = deadline;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + deadline
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createTask(String line) {
+        System.out.println(HORIZONTAL_LINE + "added: " + line + "\n" + HORIZONTAL_LINE);
+        inputs[index] = new Task(line);
+        index++;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        String line;
+
+        System.out.println(HORIZONTAL_LINE + " Hello! I'm Matthew\n" + " What can I do for you?\n" + HORIZONTAL_LINE);
+
+        while (true) {
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                break;
+            } else if (line.equals("list")){
+                System.out.println(formatList(inputs));
+            } else if (line.startsWith("mark")) {
+                mark(line);
+            } else if (line.startsWith("unmark")) {
+                unmark(line);
+            } else if (line.startsWith("todo")) {
+                createTodo(line);
+            } else if (line.startsWith("event")) {
+                createEvent(line);
+            } else if (line.startsWith("deadline")) {
+                createDeadline(line);
+            }  else {
+                createTask(line);
```

Comment:
In this case the Task and Todo classes seem functionally the same (they both only have a description, but the Todo class contains an extra marker `[T]`. You may want to consider making Todo tasks the default and remove the ability to instance the `Task` superclass with `abstract` in front of the class declaration of `Task`.